gnucash/report/reports/example/sample-report.scm

```diff
@@ -0,0 +1,458 @@
+      ;; Account Name" to html-markup-anchor.  The account name
+      ;; passed must be the "full" account name that you get from
+      ;; gnc-account-get-full-name.  You should build this url using
+      ;; (gnc-build-url ...)
+      ;;
+      ;; html-markup-anchor takes the link to jump to as its first
+      ;; arg and then puts the remaining args in the body of the
+      ;; link).
+      ;;
+      ;; html-markup-ul makes a "<ul>" unnumbered list, and takes as
+      ;; its one argument a list of items to put in <li> blocks.
+      (if (not (null? accounts))
+          (gnc:html-document-add-object!
+           document
+           (gnc:make-html-text
+            (gnc:html-markup-ul
+             (map 
+              (lambda (acct)
+                (gnc:html-markup-anchor 
+                 (gnc-build-url URL-TYPE-REGISTER
+                   (string-append "account=" 
+                     (gnc-account-get-full-name
+                        acct))
+                     "")
+                 (xaccAccountGetName acct)))
+              accounts))))
+          (gnc:html-document-add-object!
+           document
+           (gnc:make-html-text
+            (gnc:html-markup-p (G_ "You have selected no accounts.")))))
+            
+      (gnc:html-document-add-object! 
+       document 
+       (gnc:make-html-text 
+        (gnc:html-markup-anchor (gnc-build-url URL-TYPE-HELP "gnucash-guide" "") (G_ "Display help"))))
+
+      (gnc:html-document-add-object! 
+       document 
+       (gnc:make-html-text 
+        (gnc:html-markup-p (G_ "Have a nice day!"))))
+      
+      document)))
+
+;; Here we define the actual report with gnc:define-report
+(gnc:define-report
+ 
+ ;; The version of this report.
+ 'version 1
+ 
+ ;; The name of this report. This will be used, among other things,
+ ;; for making its menu item in the main menu. You need to use the
+ ;; untranslated value here!
+ 'name (N_ "Options Example Report")
+
+ ;; The GUID for this report. This string should be unique, set once
+ ;; and left alone forever after that. In theory, you could use any
+ ;; unique string, even a meaningful one (!) but its probably best to
+ ;; use a true uuid. Get them from `uuidgen | sed -e s/-//g` and paste
+ ;; the results in here. You must make a new guid for each report!
+ 'report-guid "af02e925d0484745afb04f16e0524e87"
+
+ ;; The name in the menu
+ ;; (only necessary if it differs from the name)
+ 'menu-name (N_ "Options Example")
+
+ ;; A tip that is used to provide additional information about the
+ ;; report to the user.
+ 'menu-tip (N_ "An options example report.")
+
+ ;; A path describing where to put the report in the menu system.
+ ;; In this case, it's going under the utility menu.
+ 'menu-path (list gnc:menuname-example)
+
+ ;; The options generator function defined above.
+ 'options-generator options-generator
+ 
+ ;; The rendering function defined above.
+ 'renderer options-example-renderer)
```

Comment:
The red circle means that you didn't put a newline character (from the enter/return key) at the end of the last line. Git doesn't like that.

Reply:
Thank you, that is a good tip. Now I understand I raw strips the last character and I need to replace it.

Reply:
Hopefully fixed with the most recent commit.

Reply:
Seems to be, the red circle is gone.